app/src/main/java/com/developersbreach/composeactors/core/cache/CacheManager.kt

```diff
@@ -5,27 +5,36 @@ import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
-class CacheManager @Inject constructor(
-    cacheSize: Int = 100
-) {
-    private val cache = object : LruCache<String, Any>(cacheSize) {
-        override fun sizeOf(key: String, value: Any): Int {
-            return 1
-        }
-    }
+class CacheManager
+    @Inject
```

Comment:
Multiline formatting affects readability and this is not kotlin code standards, rather we want to put injected constructor information in single line, for this we can use
```kotlin
ktlint_standard_annotation = disabled
```